src/Features/Core/Portable/Shared/Extensions/ISymbolExtensions_2.cs

```diff
@@ -84,6 +84,10 @@ public static Glyph GetGlyph(this ISymbol symbol)
                         case TypeKind.Error:
                             return Glyph.Error;
 
+                        case TypeKind.Extension:
+                            publicIcon = Glyph.ClassPublic;
```

Comment:
we may want a special glyph for extension types.  will be a UX call. this unblocks us from crashing for now.